### PR TITLE
NAS-113687 / 22.02 / Document Boot CD and Python packages

### PR DESCRIPTION
**Pull request by themylogin**

### CODE REVIEW

**Comment by themylogin** [comment]
time 1:30

File: src/middlewared/middlewared/job.py [hunk 1]
```diff
@@ -223,6 +223,9 @@ class Job:
     :ivar logs_fd: Unbuffered binary file descriptor for writing logs (if the job was defined with `@job(logs=True)`
     """
 
+    pipes: Pipes
```
Comment (yocalebo) [inline src/middlewared/middlewared/job.py:226]: Just curious, why is this change needed?

Comment (themylogin) [reply src/middlewared/middlewared/job.py:226]: Sphinx autodoc issues a warning when a documented property (`:ivar` above) is not explicitly declared. We now treat warnings as errors so...

**Review by yocalebo — APPROVED** [review]
